pyfr/readers/base.py

```diff
@@ -22,3 +26,209 @@ def to_pyfrm(self):
         mesh['mesh_uuid'] = np.array(str(uuid.uuid4()), dtype='S')
 
         return mesh
+
+
+class BaseMeshConverter(object):
```

Comment:
I removed inheritance and modified that BaseMeshConverter is constructed in raw_to_pyfrm of MeshReaders.